### ISSUE TITLE
uninstall doesnt delete secret

### ISSUE THREAD

**Issue by mattlawnz** [jenkins-x/jx#934]
Hi

after an 'jx uninstall' secrets are still left behind.
```NAME                                  TYPE                                  DATA      AGE
default-token-65z7l                   kubernetes.io/service-account-token   3         18h
jx-install-config                     Opaque                                3         16h
jx-pipeline-git-bitbucket-bitbucket   Opaque                                2         17h
jx-pipeline-git-github-github         Opaque                                2         16h```


As such subsequent installs will fail.

**Comment by andrewwippler** [comment]
I encountered this issue as well, it seems like the `jx` namespace also lingers. Removing the namespace will also remove all objects associated with it. Perhaps the cleanup job should be launched in the `default` namespace so that the `jx` namespace could be deleted?

**Comment by airwolfnh** [comment]
It also leaves behind the folder ~/.jx which will cause problems like : 
```error: Failed to create the XXXX folder in jenkins: 500 Server Error```
and
```error: error creating jenkins credential jenkins-x-chartmuseum 500 Server Error```

**Comment by ccojocar** [comment]
@rawlingsj This is fixed by #978 and #979. I tested yesterday and now the platform can be uninstall and install properly.

**Comment by rawlingsj** [comment]
Awesome @ccojocar!
